shopinvader_customer_multi_user/tests/common.py

```diff
@@ -11,59 +13,62 @@ class TestMultiUserCommon(TestCustomerCommon):
     @classmethod
     def setUpClass(cls):
```

Comment:
What to expect from this file:
- added a new common class to test partner domains, taken from the specific test file
- changed helpers to reduce the number of lines required to use them or to ease testing